migrations/versions/e1d54080707f_.py

```
@@ -0,0 +1,40 @@
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "e1d54080707f"
+down_revision: Union[str, None] = "dea241345bca"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
```

Ruff (0.8.0)

Missing docstring in public function

(D103)